modules/sdk-coin-ada/src/lib/transactionBuilder.ts

```diff
@@ -183,15 +210,47 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
         )
       );
     });
-
     if (this._changeAddress && this._senderBalance) {
       const changeAddress = CardanoWasm.Address.from_bech32(this._changeAddress);
       const utxoBalance = CardanoWasm.BigNum.from_str(this._senderBalance);
-      const change = utxoBalance.checked_sub(fee).checked_sub(totalAmountToSend);
+
+      const adjustment = 2000000;
```

Comment:
nit: we can change adjustment to already be BigNum.from_str('2000000') to reduce redundancy